Library/src/Systems/Assets/AssetSystem.cpp

```diff
@@ -385,14 +459,8 @@ void AssetSystem::UpdateAssetCollectionMetadata(const AssetCollection& AssetColl
 												const csp::common::Map<csp::common::String, csp::common::String>& NewMetadata,
 												AssetCollectionResultCallback Callback)
 {
-	csp::common::Optional<csp::common::String> SpaceId;
```

Comment:
We need this to be optional because not all AssetCollections have a space ID. Specifically, I think space thumbnails don't.

Reply:
We unilaterally agreed in the all-teams engineering meeting yesterday to represent asset collections that do not belong to a space with an empty string.

Reply:
Is this change supported in CHS yet, though? I believe we were having permission issues before, which is what led to us needing to remove space ID for certain asset types.